modules/persistent-storage-csi-secrets-store-driver-overview.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-secrets-store-driver-overview_{context}"]
+= Overview
+
+{product-title} secrets are stored with just base64 encoding, which can be easily decoded. Etcd provides encryption at rest for these secrets, but when secrets are retrieved they are decrypted and presented to the user. If RBAC is not set up on your cluster properly, anyone with API access can retrieve or modify a secret, and so can anyone with access to etcd. Additionally, anyone who is authorized to create a pod in a namespace can use that access to read any secret in that namespace. 
+
+For secure storage and management of your secrets, the {product-title} Secrets Store Container Storage Interface (CSI) Driver Operator allows you to mount secrets from external secret management system, such as AWS Secrets Manager or Azure Key Vault using a provider plugin. Applications can use the secret, but they do not persist on the system after the application pod is destroyed.
```

Comment:
I'd recommend removing "AWS Secrets Manager" as an example here, because we aren't sure yet if it will be supported for this TP release.

```suggestion
For secure storage and management of your secrets, the {product-title} Secrets Store Container Storage Interface (CSI) Driver Operator allows you to mount secrets from external secret management system, such as Azure Key Vault, using a provider plugin. Applications can use the secret, but the secret does not persist on the system after the application pod is destroyed.
```